npm/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "flow-typed",
+  "version": "2.0.0",
+  "description": "A repository of high quality flowtype definitions",
+  "main": "dist/index.js",
+  "bin": "dist/cli.js",
+  "scripts": {
+    "flow": "flow; [[ $?=0||$?=2 ]]",
+    "watch": "mkdir -p dist; babel --watch=./src --out-dir=./dist",
+    "build": "mkdir -p dist; babel ./src --out-dir=./dist",
+    "prepublish": "mkdir -p dist; babel ./src --out-dir=./dist"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/splodingsocks/FlowTyped.git"
+  },
+  "keywords": [
+    "flowtype"
+  ],
+  "author": "Dan Vanderkam (danvdk@gmail.com)",
```

Comment:
I don't, wasn't paying close attention... Was it somebody who was contributing before?